tools/clang/lib/SPIRV/LowerTypeVisitor.cpp

```diff
@@ -154,22 +162,6 @@ const SpirvType *LowerTypeVisitor::lowerType(const SpirvType *type,
         lowerType(imageAstType, rule, /*isRowMajor*/ llvm::None, loc);
     assert(isa<ImageType>(imageSpirvType));
     return spvContext.getSampledImageType(cast<ImageType>(imageSpirvType));
-  } else if (const auto *hybridFn = dyn_cast<HybridFunctionType>(type)) {
```

Comment:
Just want to doublecheck that no caller of `LowerTypeVisitor::lowerType()` passes `HybridFunctionType` now?

Reply:
The PR remove the definition of HybridFunctionType completely:
https://github.com/microsoft/DirectXShaderCompiler/pull/2620/files#diff-81cbf1d795354368cc9d08163a73b891L507-L527

So, no caller of `lowerType` can pass this type to it now.